api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/claim/ClaimService.java

```diff
@@ -50,41 +46,33 @@
             EligibilityStatus.INELIGIBLE, ClaimStatus.REJECTED
     );
 
+    // TODO HTBHF-2682 Rename this method (and others) to createClaim.
     public ClaimResult createOrUpdateClaim(ClaimRequest claimRequest) {
         try {
             EligibilityAndEntitlementDecision decision = eligibilityAndEntitlementService.evaluateNewClaimant(claimRequest.getClaimant());
-            if (decision.claimExistsAndIsEligible()) {
-                Claim claim = claimRepository.findClaim(decision.getExistingClaimId());
-                List<UpdatableClaimantField> updatedFields = updateClaim(claim, claimRequest.getClaimant());
-                sendAdditionalPaymentMessageIfNewDueDateProvided(claim, updatedFields);
-                VoucherEntitlement weeklyEntitlement = decision.getVoucherEntitlement().getFirstVoucherEntitlementForCycle();
-                claimMessageSender.sendReportClaimMessageWithUpdatedClaimantFields(claim, decision.getDateOfBirthOfChildren(), updatedFields);
-                return ClaimResult.withEntitlementAndUpdatedFields(claim, weeklyEntitlement, updatedFields, decision.getIdentityAndEligibilityResponse());
+            if (decision.getEligibilityStatus() == EligibilityStatus.DUPLICATE) {
+                Claim claim = createAndSaveClaim(claimRequest, decision);
+                claimMessageSender.sendReportClaimMessage(claim, emptyList(), REJECTED);
```

Comment:
There are a number of different enums being used in this class, can we not use any enums statically in this class as I have no idea which enum `REJECTED` is referring to without opening the code up in IntelliJ.

Reply:
Also, should this be specifically `DUPLICATE` rather than `REJECTED`?

Reply:
Updated the enums to be fully qualified. REJECTED is the claim status which doesn't have a DUPLICATE status.

Reply:
Ok thanks for clarifying.